### PR TITLE
[MM-28179] Cypress/E2E: Automate backlogs - System Console > Reporting pt. 2

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR implements [MM-T908](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T908).

#### Ticket

https://mattermost.atlassian.net/browse/MM-28179

### CODE REVIEW

**Review by isacikgoz — APPROVED** [review]
LGTM 🚀

File: e2e/cypress/integration/system_console/reporting/server_logs_spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/reporting/server_logs_spec.js:11]: Should be
```suggestion
import * as TIMEOUTS from '../../../../fixtures/timeouts';
```

File: e2e/cypress/integration/system_console/reporting/server_logs_spec.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        // # Go to the Server Logs section.
+        cy.get('#reporting\\/server_logs').click().wait(TIMEOUTS.TWO_SEC);
+    });
+
+    it('MM-T906 Logs - Verify content categories', () => {
+        // * Verify the banner is showed.
+        cy.get('.banner__content span').should('not.empty');
+
+        // * Verify reload button is showed.
+        cy.get('.admin-logs-content button span').should('be.visible').should('contain', 'Reload');
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/reporting/server_logs_spec.js:27]: nit: may use `and` for readability.
```suggestion
        cy.get('.admin-logs-content button span').should('be.visible').and('contain', 'Reload');
```

File: e2e/cypress/integration/system_console/reporting/server_logs_spec.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+        // * Verify that server logs are showed correctly.
+        cy.get('.admin-logs-content div.log__panel').should('be.visible').should('not.empty');
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/reporting/server_logs_spec.js:30]:
```suggestion
        cy.get('.admin-logs-content div.log__panel').should('be.visible').and('not.empty');
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @streamer45. LGTM, except for few comments.

**Review by saturninoabril — APPROVED** [review]
@streamer45 Thanks! LGTM, tested and passed.

**Review by lindy65 — APPROVED** [review]
Thanks @streamer45 - looks good to me 👍

**Pull request merged**